extensions/ingresses/ingresses.go

```diff
@@ -105,11 +105,23 @@ func CreateIngress(client *v1.Client, ingressName string, ingressTemplate networ
 	logrus.Infof("Create Ingress: %v", ingressName)
 	ingressResp, err := client.SteveType(IngressSteveType).Create(ingressTemplate)
 	if err != nil {
-		logrus.Errorf("Failed to create ingress: %v", err)
```

Comment:
nit; I think this log should stay for debugging.

Reply:
I can put it back, that's fine with me.

Reply:
Discussed offline, resolved.